rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/range/20_synthetic_source.yml

```diff
@@ -0,0 +1,365 @@
+setup:
+  - skip:
+      version: " - 8.13.99"
+      reason: synthetic source support added in 8.14
+
+  - do:
+      indices.create:
+        index: synthetic_source_test
+        body:
+          mappings:
+            "_source":
+              "mode": "synthetic"
+            "properties":
+              "integer_range":
+                "type" : "integer_range"
+              "long_range":
+                "type" : "long_range"
+              "float_range":
+                "type" : "float_range"
+              "double_range":
+                "type" : "double_range"
+              "date_range":
+                "type" : "date_range"
+              "ip_range":
+                "type" : "ip_range"
+
+---
+"Integer range":
+
+  - do:
```

Comment:
Thanks for putting together this exhaustive list...I think it is quite comprehensive. I was wondering if we have to test also the case where one of the bounds is null, which I think is a quite common use case...

Reply:
Great point, i have completely missed that. Thank you for the suggestion.